.github/workflows/bump.yml

```diff
@@ -21,5 +21,8 @@ jobs:
           git config --global user.name "cloud-sdk-js"
       - name: Bump version
         run: |
-          yarn changeset version
-          yarn changelog
+          yarn run version
+      - name: Commit and push
+        run: |
+          git commit -am "Increase version"
+          git push
```

Comment:
```suggestion
          currentSdkVersion = v`yarn --silent currentSdkVersion`
          git commit -am $currentSdkVersion
          git push origin `git branch --show-current`
          git tag $currentSdkVersion && git push origin $currentSdkVersion
```

Reply:
I don't know it this really works + we could also omit the remote and branch for git push.